src/client/java/teammates/client/scripts/sql/DataMigrationForCourseEntitySql.java

```diff
@@ -181,7 +181,6 @@ private void migrateStudents(teammates.storage.sqlentity.Course newCourse, teamm
             List<CourseStudent> studentsInTeam, Map<String, User> userEmailToUserMap) {
         for (CourseStudent oldStudent : studentsInTeam) {
             teammates.storage.sqlentity.Student newStudent = createStudent(newCourse, newTeam, oldStudent);
-            saveEntityDeferred(newStudent);
```

Comment:
Should be fine since no other entity relies on them

Reply:
Yeap